TYPO3.Flow/Classes/TYPO3/Flow/Object/CompileTimeObjectManager.php

```diff
@@ -206,17 +207,20 @@ protected function registerClassFiles(array $packages)
         $availableClassNames = array('' => array('DateTime'));
         /** @var \TYPO3\Flow\Package\Package $package */
         foreach ($packages as $packageKey => $package) {
-            if ($package->isObjectManagementEnabled() && (strpos($package->getComposerManifest('type'), 'typo3-flow') === 0 || isset($includeClassesConfiguration[$packageKey]))) {
-                $classFiles = $package->getClassFiles();
-                if (count($classFiles) > 0) {
-                    if (isset($this->allSettings['TYPO3']['Flow']['object']['registerFunctionalTestClasses']) && $this->allSettings['TYPO3']['Flow']['object']['registerFunctionalTestClasses'] === true) {
-                        $classFiles = array_merge($classFiles, $package->getFunctionalTestsClassFiles());
+            if ($package->isObjectManagementEnabled() && (ComposerUtility::isFlowPackageType($package->getComposerManifest('type')) || isset($includeClassesConfiguration[$packageKey]))) {
+                foreach ($package->getClassFiles() as $fullClassName => $path) {
+                    if (substr($fullClassName, -9, 9) !== 'Exception') {
```

Comment:
If I would know...
See further down in the original code. It happens there already, I just copied it over...